example_scripts/BLZ_example_plot.py

```diff
@@ -21,22 +21,27 @@
 dir_nam = "/projectsa/accord/GCOMS1k/OUTPUTS/BLZE12_02/2015/"
 fil_nam = "BLZE12_1h_20151101_20151130_grid_T.nc"
 dom_nam = "/projectsa/accord/GCOMS1k/INPUTS/BLZE12_C1/domain_cfg.nc"
+config_t = "/work/jelt/GitHub/COAsT/example_files/example_t_nemo_config.json"
```

Comment:
Before this gets merged, can we change these to something generic (i.e. using "~") and platform-independent (i.e. using Path)?